cardano-cli/src/Cardano/CLI/Shelley/Commands.hs

```diff
@@ -48,20 +49,19 @@ module Cardano.CLI.Shelley.Commands
   , Deprecated (..)
   ) where
 
-import           Prelude
-
 import           Cardano.Api.Shelley
 
-import           Data.Text (Text)
-import           Data.Time.Clock
-
+import           Cardano.Chain.Common (BlockCount)
+import           Cardano.CLI.Conway.Parsers
```

Comment:
Is there a way to avoid importing newer eras from modules for older eras? This makes reasoning what can be imported where quite hard.